### PR TITLE
LPS-185948 Automation | CP 2.0 | AssertIfCloseTheModalStatusDontChange

### PR DESCRIPTION
**Pull request by styller1**
Ticket
[LPS-185948](https://liferay.atlassian.net/browse/LPS-185948)

### CODE REVIEW

**Comment by styller1** [comment]
ci:test:sf

**Comment by styller1** [comment]
ci:test:sf

**Comment by styller1** [comment]
ci:test:sf

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/osb-site-initializer-customer-portal-test/src/testFunctional/tests/CP2LXCWithProvisioningRole.testcase [hunk 1]
```diff
@@ -45,6 +45,62 @@ definition {
...
+
+		task ("And the user goes to CP2.0 and join a project") {
+			ApplicationsMenu.gotoSite(key_site = ${testSiteName});
+
+			CustomerPortal.search(searchItem = ${testAccount});
+
+			CustomerPortal.enterOnProject(title = ${testAccount});
+		}
+
+		if (IsElementPresent(key_text = "Start Project Setup", locator1 = "Button#ANY")) {
```
Comment (renato-araujo-liferay) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/osb-site-initializer-customer-portal-test/src/testFunctional/tests/CP2LXCWithProvisioningRole.testcase]: @styller1 could you add this block of code inside a task module?

Comment (styller1) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/osb-site-initializer-customer-portal-test/src/testFunctional/tests/CP2LXCWithProvisioningRole.testcase]: Done

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/osb-site-initializer-customer-portal-test/src/testFunctional/macros/CP2Utils.macro [hunk 2]
```diff
@@ -325,6 +325,14 @@ definition {
 		Button.click(button = "Skip for now");
 	}
 
+	macro ifOnboardingPresent {
```
Comment (renato-araujo-liferay) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/osb-site-initializer-customer-portal-test/src/testFunctional/macros/CP2Utils.macro:328]: this should be remove soon as fixed by cp team.

**Review by renato-araujo-liferay — APPROVED** [review]
LGTM! Thank you @styller1

**Comment by renato-araujo-liferay** [comment]
ci:forward

**Comment by styller1** [comment]
ci:forward

**Pull request closed without merging**